interface/billing/print_daysheet_report_num1.php

```diff
@@ -68,1764 +68,1767 @@
-        print "</span></td><td width='25'><span class='text'>";
-        print "<br /></td>";
-        print "<table border='0'><tr>\n";
-        print "<td width='25'><span class='text'>";
-        print "</span></td><td width='250'><span class='text'>";
-        print "</span></td><td width='125'><span class='text'>";
-        print "</span></td><td width='250'><span class='text'>";
-        print "</span></td><td width='125'><span class='text'>";
-        print "</span></td><td width='125'><span class='text'>";
-        print "</span></td><td width='25'><span class='text'>";
-
-        $gtotal_fee = $gtotal_fee + $provider_info['fee'][$i];
-        $gtotal_insadj = $gtotal_insadj + $provider_info['insadj'][$i];
-        $gtotal_inspay = $gtotal_inspay + $provider_info['inspay'][$i];
-        $gtotal_insref = $gtotal_insref + $provider_info['insref'][$i];
-        $gtotal_patadj = $gtotal_patadj + $provider_info['patadj'][$i];
-        $gtotal_patpay = $gtotal_patpay + $provider_info['patpay'][$i];
-        $gtotal_patref = $gtotal_patref + $provider_info['patref'][$i];
-
-        ++$i;
-
-        print "<br /></td></tr>";
-    }
+            if ($totals_only === 1) {
+                $from_date = oeFormatShortDate(substr($query_part_day, 37, 10));
+                $to_date = oeFormatShortDate(substr($query_part_day, 63, 10));
+                print "<br /><br />";
+                ?><p><?php echo xlt('Totals for ') . text($from_date) . ' ' . xlt('To{{Range}}') . ' ' . text($to_date) ?></p><?php
+            }
 
-    print "<br /></td>";
-    print "</table>";
-}
-?>
+            if ($run_provider != 1) {
+                print "<table border='0'><tr>\n";
+                print "<br /><br />";
+                print  "<td width='25'><span class='text'>";
+                print  "</span></td><td width='250'><span class='text'><b><center>" . xlt("User") . "</center></b>";
+                print  "</span></td><td width='125'><span class='text'>";
+                print  "</span></td><td width='250'><span class='text'><b>" . xlt("Charges") . "</b>";
+                print  "</span></td><td width='125'><span class='text'>";
+                print  "</span></td><td width='250'><span class='text'><b>" . xlt("Payments") . "</b>";
+                print  "</span></td><td width='25'><span class='text'>";
+
+                for ($i = 1; $i < $k;) {
+                    print "<br /></td>";
+                    print "<table border='0'><tr>\n";
+                    print  "<td width='25'><span class='text'>" ;
+                    Printf("</span></td><td width='250'><span class='text'><center>" . text($user_info['user'][$i])) . "</center>";
+                    print  "</span></td><td width='125'><span class='text'>" ;
+                    printf("</span></td><td width='250'><span class='text'><b>" . xlt("Total Charges") . ': ' . " %1\$.2f ", text($user_info['fee'][$i])) . "</b>";
```

Comment:
Hi @bradymiller  and @tywrenn 

I think this page needs to do a lot refine, should I finish it on this PR?
and convert a lot of `print` function?
Also, in the L-1700 contains the `Print` function, is it correct?

Reply:
@stu01509 You should definitely do this all in this PR. I'm not sure what you mean by converting the print function - please explain.

Reply:
I mean this page using the print function to render the view, but I think it is unnecessary,  I would like to convert to pure HTML.

Reply:
@stu01509 Yeah you can go ahead and do that